packages/brand-positioning-block/src/components/BlockInjectButton.tsx

```diff
@@ -0,0 +1,164 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    ActionMenu,
+    Flyout,
+    IconArrowCircleUp20,
+    IconImageStack20,
+    LoadingCircle,
+    MenuItemContentSize,
+} from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import React, { useEffect, useRef, useState } from 'react';
+import { BlockInjectButtonProps } from '../types';
+
+const BlockInjectButton = ({
+    onClick,
+    onDrop,
+    label,
+    icon,
+    secondaryLabel,
+    isLoading,
+    onAssetChooseClick,
+    onUploadClick,
+}: BlockInjectButtonProps) => {
+    const [isDraggingOver, setIsDraggingOver] = useState(false);
+    const [menuPosition, setMenuPosition] = useState<[number, number] | undefined>();
+    const buttonRef = useRef<HTMLButtonElement>(null);
+    useEffect(() => {
+        const handleClickOutside = (event: MouseEvent) => {
+            if (buttonRef.current && !buttonRef.current.contains(event.target as HTMLElement)) {
+                setMenuPosition(undefined);
+            }
+        };
+        document.addEventListener('mousedown', handleClickOutside);
+        return () => {
+            document.removeEventListener('mousedown', handleClickOutside);
+        };
+    }, []);
+
+    const handleDrop: React.DragEventHandler<HTMLButtonElement> = (e) => {
+        e.preventDefault();
+        setIsDraggingOver(false);
+        if (onDrop) {
+            const dt = e.dataTransfer;
+            const files = dt.files;
+            onDrop(files);
+        }
+    };
+
+    const openMenu: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+        if (!buttonRef.current || isLoading) {
+            return;
+        }
+        const { left, top } = buttonRef.current.getBoundingClientRect();
+        const XInsideComponent = e.clientX - left;
+        const YInsideComponent = e.clientY - top;
+        setMenuPosition([XInsideComponent, YInsideComponent]);
+    };
+
+    return (
+        <button
+            ref={buttonRef}
+            className={joinClassNames([
+                ' tw-font-body tw-relative tw-h-[72px] tw-text-[14px] tw-text-text-weak tw-border tw-rounded-[4px] tw-flex tw-items-center tw-justify-center tw-cursor-pointer tw-gap-3 tw-w-full',
```

Comment:
Remove the space at the start of the string and:
- `tw-text-[14px]` -> `tw-text-sm`
- `tw-rounded-[4px]` -> `tw-rounded`
Not 100% on these, but we should have values of those sizes.